astropy/wcs/tests/test_wcs.py

```diff
@@ -721,3 +726,17 @@ class NewWCS(wcs.WCS):
     with pytest.raises(TypeError) as exc:
         iter(w)
     assert exc.value.args[0] == "'NewWCS' object is not iterable"
+
+
+def test_sip_tpv_agreement():
+    sip_header = get_pkg_data_contents(
+        os.path.join("data", "siponly.hdr"), encoding='binary')
+    tpv_header = get_pkg_data_contents(
+        os.path.join("data", "tpvonly.hdr"), encoding='binary')
+
+    w_sip = wcs.WCS(sip_header)
+    w_tpv = wcs.WCS(tpv_header)
+
+    assert_array_almost_equal(
+        w_sip.all_pix2world([w_sip.wcs.crpix], 1),
+        w_tpv.all_pix2world([w_tpv.wcs.crpix], 1))
```

Comment:
@mdboom I read how TPV is defined here http://iraf.noao.edu/projects/ccdmosaic/tpv.html and just for my  understanding - I'm wondering why this is a valid test, given that the order of operations is different (in the WCS paper language one is prior, the other sequent). 

It is possible to convert from one type to the other. Assuming wcslib converts them internally, were the TPV coefficients in this test created from the SIP coefficients and so basically this tests the agreement between astropy.wcs.sip and wcslib sip? Or am I misunderstanding it?